### ISSUE TITLE
Tool does not write errors to stderr

### ISSUE THREAD

**Issue by mlafeldt** [aliyun/aliyun-cli#266]
While working on https://github.com/mlafeldt/dotfiles/blob/master/bin/alicloud-assume-role, I noticed that the `aliyun-cli` tool does not write errors to stderr. Instead all output is written to stdout.

Proof:

```
$ aliyun sts AssumeRole >/dev/null

$ aliyun sts AssumeRole 2>/dev/null
ERROR: required parameters not assigned:
  --RoleArn
  --RoleSessionName
```

This [makes scripting harder](https://github.com/mlafeldt/dotfiles/blob/524ff1c0d31ecdc9f9626af7536b247389d6b830/bin/alicloud-assume-role#L45-L47) than it should be.

**Comment by night556** [comment]
> While working on https://github.com/mlafeldt/dotfiles/blob/master/bin/alicloud-assume-role, I noticed that the `aliyun-cli` tool does not write errors to stderr. Instead all output is written to stdout.
> 
> Proof:
> 
> ```
> $ aliyun sts AssumeRole >/dev/null
> 
> $ aliyun sts AssumeRole 2>/dev/null
> ERROR: required parameters not assigned:
>   --RoleArn
>   --RoleSessionName
> ```
> 
> This [makes scripting harder](https://github.com/mlafeldt/dotfiles/blob/524ff1c0d31ecdc9f9626af7536b247389d6b830/bin/alicloud-assume-role#L45-L47) than it should be.

Thank you for your issue, this issue is a legacy of history and we are currently repairing it. We will reply under this issue after the repair.

**Comment by aschleifer** [comment]
Is there any update on when this issue might get fixed? It makes it annoying to debug issues in automation if this is not fixed.

**Comment by JacksonTian** [comment]
We will release a version to fix it before 7.31.

**Comment by JacksonTian** [comment]
After #373 merged and release new CLI, this issue will be fixed.
